src/main/java/com/ita/if103java/ims/dao/impl/ItemDaoImpl.java

```diff
@@ -73,33 +77,35 @@ public Item findItemById(Long id) {
     @Override
     public Item addItem(Item item) {
         try {
-            jdbcTemplate.update(Queries.SQL_INSERT_INTO_ITEM, item.getName(), item.getUnit(), item.getDescription(), item.getVolume(), item.isActive(), item.getAccountId());
+            System.out.println(item.getUnit());
```

Comment:
Please pay more attention on what are you pushing. 
Remove System.out.println(item.getUnit()) or replace with logger